lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -1085,7 +1085,8 @@ class SPIRVSwitch : public SPIRVInstruction {
     return static_cast<SPIRVBasicBlock *>(getValue(Default));
   }
   size_t getLiteralsCount() const {
-    return getSelect()->getType()->getBitWidth() / (sizeof(SPIRVWord) * 8);
+    unsigned ByteWidth = getSelect()->getType()->getBitWidth() / 8;
+    return (ByteWidth / sizeof(SPIRVWord)) + !!(ByteWidth % sizeof(SPIRVWord));
```

Comment:
I think it would be more readable if rewrite this `!!(ByteWidth % sizeof(SPIRVWord))` into `((ByteWitdh % sizeof(SPIRVWord)) != 0)`

Reply:
Good point, perhaps it's even better to factor the modulo expression out into a separate variable.